vcf-dumper/vcf-dumper-cli/src/main/java/uk/ac/ebi/eva/vcfdump/VariantExportBootApplication.java

```diff
@@ -22,6 +22,7 @@
 import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import uk.ac.ebi.eva.vcfdump.VariantExporterController;
```

Comment:
Do you really need this commit, because of the changes in the pom.xml? Or did you forget to remove it after the tests you were doing because maven install didn't work?